### PR TITLE
[SPARK-27201][WebUI] Toggle full job description on click

### PR DESCRIPTION
**Pull request by gengliangwang**
## What changes were proposed in this pull request?

Previously, in https://github.com/apache/spark/pull/6646 there was an improvement to show full job description after double clicks.
I think this is a bit hard to be noticed by some users. I suggest changing the event to one click.
Also, after the full description is shown, another click should be able to hide the overflow text again.

Before click:
short

After click:
full

Click again:
short

## How was this patch tested?

Manually check.

### CODE REVIEW

**Comment by gengliangwang** [comment]
@wangyum @srowen

File: core/src/main/resources/org/apache/spark/ui/static/additional-metrics.js [hunk 1]
```diff
@@ -84,7 +84,7 @@ $(function() {
     });
 
     // Trigger a double click on the span to show full job description.
```
Comment (wangyum) [inline core/src/main/resources/org/apache/spark/ui/static/additional-metrics.js]: Please update the comment.

**Comment by srowen** [comment]
I'm OK with that. Does it still allow you to click the link in the cell? just want to make sure it doesn't interfere with single clicks on the hyperlink.

**Comment by gengliangwang** [comment]
@srowen Yes, the link still works.

**Review by felixcheung — APPROVED** [review]
SGTM

**Comment by dongjoon-hyun** [comment]
Retest this please.

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM (Pending Jenkins). This is much nicer than before. Previously, it's only one-way (expanding only). Now, it's two-way (expand and collapse by toggle). Thanks, @gengliangwang !

**Comment by wangyum** [comment]
retest this please

**Review by HyukjinKwon — APPROVED** [review]

**Comment by HyukjinKwon** [comment]
Merged to master.

**Pull request closed without merging**